README.md

```diff
@@ -22,9 +22,9 @@ From Amplitude Flutter v4, we bump up the kotlin version to v1.9.22 to support l
 
 The following matrix lists the minimum support for Amplitude Flutter SDK version.
 
-| Flutter | Amplitude Flutter | Gradle | Android Gradle Plugin | Kotlin Gradle Plugin |
-|---------|-------------------|-------|-----------------------|-----------------------|
-|`3.3+`   | `4.+`             | `8.2` | `8.2.2`               | `1.9.22`              |
+|  Dart   | Flutter | Amplitude Flutter | Gradle | Android Gradle Plugin | Kotlin Gradle Plugin |
+|---------|---------|-------------------|-------|-----------------------|-----------------------|
+| `3.7.0+`|`3.3+`   | `4.+`             | `8.2` | `8.2.2`               | `1.9.22`              |
 
```

Comment:
What requires Dart version? `sharedDarwinSource` requires Flutter 3.7 right?

Reply:
Ah, good catch. Yes, Flutter requires 3.7 for `sharedDarwinSource`, but we need dart 3.3+ to support Flutter web after adopting `js_interop`. I'll swap those around